src/SwedbankPay/Api/Client/ClientVersion.php

```diff
@@ -143,6 +143,9 @@ private function tryGetVersionNumberFromComposerJson(&$version) : bool
     private function tryGetVersionNumberFromComposerLock(&$version) : bool
     {
         $composerLockPath = getcwd() . DIRECTORY_SEPARATOR . 'composer.lock';
+        if (!file_exists($composerLockPath)) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The use of function file_exists() is discouraged](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-php/pullRequest?prid=5523738)